common/app/routes/challenges/redux/step-challenge-epic.test.js

```diff
@@ -87,7 +87,7 @@ test(file, function(t) {
         }
       };
     });
-    stepChallengeEpic(actions, () => state)
+    stepChallengeEpic(actions, { getState: () => state })
```

Comment:
Always keep PR's scoped to the specific change you are introducing. I understand why this change is here, but it adds a lot of noise to the PR that makes it hard to parse your specific changes.